### DISCUSSION TITLE
Wrong fans speed

### DISCUSSION THREAD

**Discussion by tonicb78** [LibreHardwareMonitor/LibreHardwareMonitor#381] [General]
First of all, thanks a lot for this nice project, very useful software.

I have an Asus TUF GAMING B450-PLUS II and the fans are showing unreal reads, some very low (like 10rpm) and others very high (5000rpms):

|  |  +- Fan #1         :  20.7361  20.7322  20.7386 (/lpc/it8665e/fan/0)    ---> CPU
|  |  +- Fan #2         :  3813.56  3770.95  3813.56 (/lpc/it8665e/fan/1)
|  |  +- Fan #3         :     3000  2986.73  3013.39 (/lpc/it8665e/fan/2)
|  |  +- Fan #4         :   639.81   639.81  640.418 (/lpc/it8665e/fan/3)

HWMonitor or other related softwares seems to be able to read them properly:

	Fan 0			784 RPM [0x35D] (CPU)
	Fan 1			584 RPM [0x484] (FANIN1)
	Fan 2			560 RPM [0x4B5] (FANIN2)
	Fan 3			655 RPM [0x407] (FANIN3)

Let me know if I can help.

Thanks a lot!

**Comment by FlyingHippo** [comment]
I have the same motherboard. My casefans are connected to "Fan #4" and works perfectly, i can both read and control the RPM.
My CPU fan is connected to "Fan #0" and are showing 21RPM
